docs/src/ann_indexes.md

```diff
@@ -0,0 +1,59 @@
+# ANN (Approximate Nearest Neighbor) Indexes
+
+You can create an index over your vector data to make search faster. Vector indexes are faster but less 
+ accurate than exhaustive search. LanceDB provides many parameters to fine-tune the index's size, the speed of 
+queries, and the accuracy of results.
```

Comment:
Let's add something like "Currently, LanceDB does not automatically create the ANN index. In the future we will look to improve this experience and automate index creation and configuration"